### ISSUE TITLE
view product

### ISSUE THREAD

**Issue by vitormcsi** [nordcomputer/magento2-showoutofstockprice#23]
hi, to can add option to display price in view product? Thanks

**Comment by nordcomputer** [comment]
Sorry, can you rephrase your question? I dont really understand it. The price should be visible on the product page.

**Comment by vitormcsi** [comment]
Hi, in magento 2.3.7-p2, when I have the state in out of stock, the price of the product does not appear either in the product view or in the catalog view.

**Comment by nordcomputer** [comment]
- Did you switch on the configuration for the Extension? (Shops -> Configuration -> Nordcomputer -> Show prices for out-of-stock products)

**Comment by vitormcsi** [comment]
Hi, thanks for responding so quickly.
I have the following options:
In General Configuration
Show swatches for out-of-stock prodcts on product page - Yes
Show Add to cart button on product page - No
Page Category
Show swatches even if all products are out-of-stock - Yes
Show add to cart button even if all products are out-of-stock - No
Search
Show swatches even if all products are out-of-stock - Yes
Show add to cart button even if all products are out-of-stock - No

Is there any Show prices for out-of-stock products option of yes or no? I don't at least.
Regards

**Comment by nordcomputer** [comment]
There are some things, that should be set (outside of the extension settings):
To show the swatches, the swatch-attribute must actually be a swatch-attribute (not a dropdown).
Also, the attribute must be set to be "Used in product listing" as true.

But the price should be visible in all cases.

category-page:
![grafik](https://user-images.githubusercontent.com/13378398/179091625-302d83b6-2d57-40f4-8187-3ff983265de5.png)

product.page:
![grafik](https://user-images.githubusercontent.com/13378398/179092359-c3ce6c7f-308e-4d36-92c4-eaa217e48398.png)

configuration:
![grafik](https://user-images.githubusercontent.com/13378398/179091676-c989b674-0d8c-4e8b-b183-d161c35e92ca.png)

Have you tried to switch to the Luma-theme, as it may be theme-related?

**Comment by vitormcsi** [comment]
Hi, the swatches are working fine, it just doesn't show me the price. I tested it as the Luma theme and it is not shown either. Is it possible that the cause is from magento version 2.3.7-p2? I've been looking closely at other extensions to check for any conflicts and I can't find them.
The plugin is correctly activated, otherwise it wouldn't show up in swatches. Regards

**Comment by nordcomputer** [comment]
I am still not able to reproduce your issue - have you tried to use it in a clean (new) magento installation?

**Comment by nordcomputer** [comment]
As there seems to be no further information, I close this issue - feel free to reopen, if it is still present.
